paper-server/src/main/java/io/papermc/paper/configuration/WorldConfiguration.java

```diff
@@ -171,6 +171,7 @@ public class Sniffer extends ConfigurationPart {
         public class Spawning extends ConfigurationPart {
             public ArrowDespawnRate nonPlayerArrowDespawnRate = ArrowDespawnRate.def(WorldConfiguration.this.spigotConfig);
             public ArrowDespawnRate creativeArrowDespawnRate = ArrowDespawnRate.def(WorldConfiguration.this.spigotConfig);
+            public int arrowDespawnOverrideRate = 200;
```

Comment:
Should be an IntOr.Disabled if you want to support not despawning at all via this.